### PR TITLE
Updated `Babel` package version

### PR DESCRIPTION
**Pull request by VJalili**
Updated `Babel` package version to meet minimum requirements of the `CloudBridge` package.

### CODE REVIEW

File: wheels/build/wheels.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ purepy_packages:
     azure-storage:
         version: 0.32.0
     Babel:
-        version: 2.1.1
+        version: 2.3.4
```
Comment (nsoranzo) [inline wheels/build/wheels.yml]: I'd update to the latest release 2.4.0 unless there's a specific reason.

Comment (VJalili) [reply wheels/build/wheels.yml]: @nsoranzo Thanks, updated to the latest version.

**Comment by natefoo** [comment]
Here's the wheel for testing: https://depot.galaxyproject.org/starforge/wheels/build-192/

**Comment by VJalili** [comment]
tested, and it works fine; thanks @natefoo

**Comment by natefoo** [comment]
Ok, it's been pushed to wheels.galaxyproject.org

**Comment by VJalili** [comment]
Thanks.

**Pull request merged**